STX.REST.RESTFulSense.Clients/Brokers/Https/HttpBroker.cs

```diff
@@ -0,0 +1,35 @@
+﻿// ----------------------------------------------------------------------------------
+// Copyright (c) The Standard Organization: A coalition of the Good-Hearted Engineers
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace STX.REST.RESTFulSense.Clients.Brokers.Https
+{
+    internal partial class HttpBroker : HttpClient, IHttpBroker
+    {
+        private readonly HttpClient httpClient;
+
+        public HttpBroker() { }
+
+        public HttpBroker(HttpClient httpClient)
+        {
+            this.httpClient = httpClient;
+        }
+
+        private static async ValueTask<T> DeserializeResponseContent<T>(
```

Comment:
I think this method should not exist here because the deserialization/serialization is done under is own service.
Then, no sequencing allowed under brokers. No ifs, no iterations....